src/Wallet.tsx

```diff
@@ -280,7 +289,7 @@ const Wallet: React.FunctionComponent<{ role: Role }> = (props: {
                     });
                 }}
               >
-                <BoltIcon /> L2 Pay
+                <BoltIcon /> Bridge to {myPeerChain.symbol}
```

Comment:
Could use the "name" field here, although that is typically a little too long to make the button look good...

We could just use "bridge" in the button, and display the chain name (and/or logo?) somewhere else on the UI?

Reply:
i.e. Host Network / Target Network